services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -342,6 +344,15 @@ async def update_projects_networks(
     )
 
 
+def _is_service_inactive(activity_info: ActivityInfoOrNone, threshold: float) -> bool:
```

Comment:
MINOR: just to be more precise. The idea here is to flag the service when it is too long without activity ie.inactive. Strictly speaking a service is inactive when `activity_info.seconds_inactive>0`. 
Some suggestions by chatty and tuned by me ... 

```python
def is_service_inactive_beyond_threshold( ... ,  inactivity_threshold: float) -> bool:
  ...
  
def is_service_long_inactive( ... , inactivity_threshold: float) -> bool:
    ...
    
def has_service_been_inactive_too_long( ... , max_allowed_inactivity: float) -> bool:
  ...

```